app/public/model/UserPreferences.js

```diff
@@ -39,8 +39,9 @@ export default class UserPreferences extends Observable {
         this.detectorList = RCT.detectors.reduce((acc, detector) => ({ ...acc, [detector]: true }), {});
     }
 
-    setRowsOnSite(rowsOnSite) {
-        this.rowsOnSite = rowsOnSite;
+    setItemsPerPage(itemsPerPage) {
+        this.rowsOnSite = itemsPerPage;
```

Comment:
Some time ago we were told to change `page` to `site` (prev version: `rowsPerPage`, now `rowsOnSite`), but in the BK I've seen `page` being used in this context everywhere. Should I change all the related variables this way?

Reply:
The lines below need to stay for retro-compatibility reasons (other views still need them for changing `rowsOnSite`)